docs/installation/est/Installing_EST.md

```diff
@@ -1,14 +1,11 @@
-Installing EST
-==============
+# Installing EST
 
-Overview
---------
+## Overview
 
 This page describes the process to install a *EST subsystem*.
 
 
-Prerequisite
---------------------------
+## Prerequisite
 
 Create a Dogtag user group for EST RA accounts (**EST RA Agents**), and an EST RA
```

Comment:
To avoid confusion, I'd start with "On the CA," so it reads "On the CA, create a Dogtag user group for the EST RA accounts ..."

Reply:
Also, for the pki commands below, it's best not to assume the defaults on user's command line, because if they happen to use something different, it might frustrate them to have to dig up the information on their own.  Perhaps add a note at the top that says something like "The commands below assumes that the default host and port number of the CA are the local host and 8443 (correct me if I'm wrong).  To specify a different host and port number, use -h and -p respectively.  It is also assumed that the nssdb is under ~/.dogtag/nssdb (to reflect @edewata 's suggestion above).  To specify a different nssdb path, use -d"

Reply:
Also (sorry I can't seem to add comment to the exact line below) for the EST profile, estServerCert.cfg already exists in the CA, so we just need to enable it (maybe with some edits, such as AIA), isn't it?

Reply:
@ladycfu I think you are right although in all the tests I have done I always add the profile and do not get any warning like "The profile already exist!". I'll remove the add and double check with the CI.

Reply:
@ladycfu I did a test and it is not included in the default profile available with the CA so it need to be added explicitly.  At least using the master branch and I think it should be the same for other branches so I am leaving this step.